### PR TITLE
updated README.md for the next tag

### PR DESCRIPTION
**Pull request by smalti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,25 +251,32 @@ Here are some examples of library usage: [gmock-win32-sample](https://github.com
 
 # Version history
 
+## Version 1.2.0 (** October 2023)
```
Comment (smalti) [inline README.md:254]: It may be necessary to create a new version, as there have been significant changes.

Comment (Alex0vSky) [reply README.md:254]: Feel joy when a new version appears. A lot of work has been done.

**Review by Alex0vSky — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (smalti) [reply README.md]: Yep, thanks for contributing!

**Review by Alex0vSky — APPROVED** [review]
Later we can make changes to the documentation section -- "Fixed problem with Windows ApiSet DLL functions redirection". For search engines so people can find them, keywords.
There was a problem with an earlier version of `gmock-win32`.

**Pull request merged**